lib/drives/pages/drive_detail_page.dart

```diff
@@ -231,16 +231,16 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
       ),
     );
 
-    final startIcon = Icon(Icons.north_east_rounded, color: Colors.green.shade700);
-    const endIcon = Icon(Icons.south_west_rounded, color: Colors.red);
+    final startIcon = Icon(Icons.north_east_rounded, color: Theme.of(context).own().success);
+    final endIcon = Icon(Icons.south_west_rounded, color: Theme.of(context).errorColor);
     for (int index = 0, length = stop.actions.length; index < length; index++) {
       final action = stop.actions[index];
       final icon = action.isStart ? startIcon : endIcon;
       final profile = action.profile;
 
       Widget container = Container(
         decoration: BoxDecoration(
-          color: Colors.grey.withOpacity(0.3),
+          color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5),
```

Comment:
```suggestion
          color: Theme.of(context).colorScheme.onSurface.withOpacity(0.2),
```
I think it looks better if this is less than 0.5

Reply:
Yes that was a mistake! Looks like it should actually be 0.1 when I compare it with how it was before